inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -10,6 +10,14 @@
 std::vector<std::string> disabledTestPatterns() {
     return {
         // TODO: FIX BUG 31661
-        ".*Behavior.*CallbackThrowException.*"
+        ".*Behavior.*CallbackThrowException.*",
+        // TODO: Issue 32541
+        R"(.*(EltwiseLayerTest).*eltwiseOpType=Prod.*secondaryInputType=PARAMETER.*)",
```

Comment:
change to R"(.*(EltwiseLayerTest).*eltwiseOpType=Prod.*secondaryInputType=PARAMETER.*Type=SCALAR.*)",